requirements/edx/github.txt

```diff
@@ -80,7 +80,7 @@ git+https://github.com/edx/edx-ora2.git@1.1.5#egg=ora2==1.1.5
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
 git+https://github.com/edx/i18n-tools.git@v0.3.1#egg=i18n-tools==v0.3.1
 git+https://github.com/edx/edx-val.git@0.0.9#egg=edxval==0.0.9
--e git+https://github.com/pmitros/RecommenderXBlock.git@518234bc354edbfc2651b9e534ddb54f96080779#egg=recommender-xblock
+-e git+https://github.com/pmitros/RecommenderXBlock.git@v1.0#egg=recommender-xblock==1.0
```

Comment:
We'll need to update the version number here after https://github.com/pmitros/RecommenderXBlock/pull/43 merges